app/assets/javascripts/advanced-search-results-table.js

```diff
@@ -6,9 +6,8 @@ function cleanColumnOutput (data, type, row) {
 $(document).ready(function () {
   const searchQuery = decodeURIComponent(window.location.search.substring(1))
 
-  const search = JSON.parse('{"' + decodeURI(searchQuery).replace(/"/g, '\\"').replace(/&/g, '","').replace(/=/g, '":"').replace(/\+/g, ' ') + '"}')
-
   if (searchQuery) {
+    const search = JSON.parse('{"' + decodeURI(searchQuery).replace(/"/g, '\\"').replace(/&/g, '","').replace(/=/g, '":"').replace(/\+/g, ' ') + '"}')
```

Comment:
This is not actually part of this PR however I spotted it was generating a JavaScript error if there is no search query.